### DISCUSSION TITLE
How to sync ratings from IMDB to TRAKT

### DISCUSSION THREAD

**Discussion by djharborne** [cecobask/imdb-trakt-sync#23] [Q&A] [answered]
This workflow should allow me to export ratings from my IMBD account and import them into TRAKT. The workflow has run several times without any issues, but there have not been any noticeable changes in TRAKT.

I have grabbed the cookie information from IMDB and my login details for TRAKT, so everything should be working. Have I missed something?

**Comment by cecobask** [comment, marked as answer]
Hi @djharborne,

By default the syncer runs in `dry-run` mode to prevent undesired changes to your Trakt data. It informs you what changes it would have done.

If you’d like to get it to sync your resources, you have to update the `SYNC_MODE` variable to one of the values documented here: https://github.com/cecobask/imdb-trakt-sync/blob/8bd8f8da2c9b88b61a5c96a1ba91416035a269d6/.env.example#L39-L45

**Reply by djharborne** [reply]
Oh, how did I miss that? Beautiful! All working now.
